### PR TITLE
Modify Elasticsearch Daemon Setup

### PR DESCRIPTION
**Pull request by ppp0**
The provided args needed to be split into
* Java Args (Heap etc.) -> exported env vars
* Elasticsearch args -> weaved into daemon args

### CODE REVIEW

**Comment by ppp0** [comment]
depends on #1482  (though already merged in this PR)

**Comment by ppp0** [comment]
@kris-lab @njam anyone up for reviewing?

**Comment by njam** [comment]
@kris-lab I'll leave this one for you, sorry :)

File: modules/daemon/templates/sysvinit.sh.erb [hunk 1]
```diff
@@ -16,7 +16,9 @@ DAEMON_ARGS='<%= @args %>'
 PIDFILE='/var/run/<%= @name %>.pid'
 USER='<%= @user %>'
 <% unless @env.empty? %>
-export<% @env.each do |name, value| -%> <%= name %>='<%= value %>'<% end -%>
+<% @env.each do |name, value| %>
+export <%= name %>='<%= value %>'
+<% end %>
```
Comment (kris-lab) [inline modules/daemon/templates/sysvinit.sh.erb]: Let's verify if there is no empty lines?

**Comment by kris-lab** [comment]
I added small comment in the code. The rest looks good to me.

**Pull request merged**